doc/stdenv/stdenv.chapter.md

```diff
@@ -373,7 +373,7 @@ Additional file types can be supported by setting the `unpackCmd` variable (see
 
 ##### `srcs` / `src` {#var-stdenv-src}
 
-The list of source files or directories to be unpacked or copied. One of these must be set.
+The list of source files or directories to be unpacked or copied. One of these must be set. Note that if you use `srcs`, you should also set `sourceRoot` or `setSourceRoot`.
```

Comment:
Set it to what? Not even `sourceRoot` makes clear where the names come from. I could make a guess, but if my guess is right, `srcs` is badly designed and should be an attrset instead of a list, but I guess that's an improvement for another time.

Reply:
Sorry for the tone just there. Nothing personal. In fact I really appreciate your doc PR!

Reply:
Haha, it's true. I'm not even really sure how to use this, I just set it to `"."`. You are probably right, and the generic builder should do something more intelligent with `srcs` / `sourceRoot`. But I don't have the time to figure this out, and writing a PR to the docs was easier :sweat_smile: